zcash_client_backend/src/data_api.rs

```diff
@@ -1,5 +1,6 @@
 //! Interfaces for wallet data persistence & low-level wallet utilities.
 
+use secrecy::SecretVec;
```

Comment:
This is in the wrong import block for our convention. Move it to the same block as `zcash_primitives` (or a new block immediately above it, but from a dependency perspective `secrecy` and `zcash_primitives` are both crates.io dependencies).